.dockerignore

```diff
@@ -0,0 +1,13 @@
+__pycache__/
+*.DS_Store*
+**/.env
+*.idea/*
+dataset
+lightning_logs/
+notebooks/
+dist/*
+tests/test_examples/test_runs_via_python/mock_dataset/
+tests/test_examples/test_runs_via_configs/logs/
+mock_dataset/
```

Comment:
tests/test_examples/test_runs_via_python/logs/
tests/test_examples/test_runs_via_configs/mock_dataset/
?

Reply:
add it to gitignore as well

Reply:
copied from .gitignore
already fixed in another PR